test/app/handler/month/creation.spec.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+var MonthCreation = require('handler/month/creation');
+var TZDate = require('common/timezone').Date;
+
+describe('MonthCreation', function() {
+    var proto;
+
+    beforeEach(function() {
+        proto = MonthCreation.prototype;
```

Comment:
Is there a reason for using MonthCreation.prototype?
I think it's better to use MonthCreation instance.

Reply:
I just want to test `MonthCreation.prototype._adjustStartAndEndTime` and focus on it. And in many cases instantiation has more cost than just using prototype.  Because when you need instance, you can use `call` function with only meaningful parameters. 
What are you think about this?

Reply:
OK. Prototype seems to be more efficient.